fileicons/openhab-icon-theme.json

```diff
@@ -0,0 +1,150 @@
+{
+    "iconDefinitions": {
+        "_file": {
+            "iconPath": "./images/File_16x.svg"
+        },
+        "_folder": {
+            "iconPath": "./images/Folder_16x.svg"
+        },
+        "_folder-open": {
+            "iconPath": "./images/FolderOpen_16x.svg"
+        },
+        "_file_configure": {
+            "iconPath": "./images/File_16x_Configure.png"
```

Comment:
Yes, you are right. It would be better to have svg icons. But I could not find any svg icons from the Silk Icon Package, which is used on the ESH Designer.